src/components/MovieFilter/MovieFilter.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from './MovieFilter.module.scss';
+import { useState } from 'react';
+import classNames from 'classnames';
+
+const valueFilter = ['All', 'Documentary', 'Comedy', 'Horror', 'Crime'];
+
+export const MovieFilter = () => {
+  const [activeGenre, setActiveGenre] = useState(valueFilter[0]);
+
+  const onHandleGenre = (genre: string) => {
+    setActiveGenre(genre);
+  };
+
+  return (
+    <div className={styles.movieFilter}>
+      {valueFilter.map((genre) => (
+        <button
+          key={genre}
+          className={classNames(
+            `${styles.movieFilterBtn}`,
+            genre === activeGenre ? `${styles.movieFilterBtnChosen}` : '',
+          )}
+          onClick={() => onHandleGenre(genre)}
```

Comment:
ditto

Reply:
![image](https://user-images.githubusercontent.com/73526360/171435153-8cb30815-4155-4359-91c3-613286747789.png)
is it good or not?